--- ./ryu/ryu/app/simple_switch_13.py
@@ -46,20 +46,25 @@ class SimpleSwitch13(app_manager.RyuApp):
         actions = [parser.OFPActionOutput(ofproto.OFPP_CONTROLLER,
                                           ofproto.OFPCML_NO_BUFFER)]
         self.add_flow(datapath, 0, match, actions)
+        self.add_flow(datapath, 1, parser.OFPMatch(eth_type=0x88cc), [])

     def add_flow(self, datapath, priority, match, actions, buffer_id=None):
         ofproto = datapath.ofproto
         parser = datapath.ofproto_parser
-
+       idle_timeout = 30
+       hard_timeout = 0
         inst = [parser.OFPInstructionActions(ofproto.OFPIT_APPLY_ACTIONS,
                                              actions)]
         if buffer_id:
             mod = parser.OFPFlowMod(datapath=datapath, buffer_id=buffer_id,
-                                    priority=priority, match=match,
-                                    instructions=inst)
+                                    idle_timeout=idle_timeout, hard_timeout=hard_timeout,
+                                       priority=priority, match=match,
+                                    instructions=inst, table_id=100)
         else:
             mod = parser.OFPFlowMod(datapath=datapath, priority=priority,
-                                    match=match, instructions=inst)
+                                   idle_timeout=idle_timeout,
+                               hard_timeout=hard_timeout, match=match,
+                                       instructions=inst, table_id=100)
         datapath.send_msg(mod)

     @set_ev_cls(ofp_event.EventOFPPacketIn, MAIN_DISPATCHER)

